packages/donations-app/app/components/dashboard/charityDashboardController.js

```diff
@@ -20,20 +20,39 @@ angular.module('aliceApp')
             vm.project = result.data.current_project;
           }
           vm.general_outcomes = vm.projectWithGoals.goals;
-          vm.donated_outcomes = _.map(vm.projectWithGoals.donated, function(item) {
-            return _.extend(item, _.findWhere(vm.projectWithGoals.validated, { _id: item._id }));
+
+          vm.projectWithGoals.validated.forEach((validatedOutcome) => {
+            let outcome = vm.general_outcomes.find((outcome) => {
+              if(outcome._id === validatedOutcome._id) {
```

Comment:
Find just need a boolean value as return, anyway the
 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some
will suit it better. 
Could we rename it to "hasValidatedOutcome" to be clear what we're doing?